source/d_geometryblocks.rst

```diff
@@ -0,0 +1,190 @@
+
+In the pre-compute you have to provide some sort of geographic bounding area.
+This can be through a predefined boundary in Lizard (like in the example) or a geometric bounding box.
+The start parameter applies a temporal filter to the features included.
+Features which are valid at the start date are computed, features which were built after or removed before the start-date are not computed. 
+
+Label endpoint
+--------------
+
+Labels which have been pre-computed are stored in the labels endpoint of the API: https://demo.lizard.net/api/v3/labels/.
+By using this endpoint it is possible to request both individual labels and label statistics. Through a GET request it is possible to determine statistics of the entire labeltype or specific regions: https://demo.lizard.net/api/v3/labels/counts/?label_type__uuid={label type uuid}. 
+
+Labels which have been pre-computed are stored in the labels endpoint of the API: https://demo.lizard.net/api/v3/labels/.
+By using this endpoint it is possible to request both individual labels and label statistics. Through a GET request it is possible to determine statistics of the entire labeltype or specific regions: https://demo.lizard.net/api/v3/labels/counts/?label_type__uuid={label type uuid}. 
+
+Examples of a graph
+-------------------
+
+Geometry example: classify build year of buildings.
+
+.. code-block:: json
+
+    {
+        "source": {
+            "graph": {
+                "buildings": [
+                    "geoblocks.geometry.sources.GeoDjangoSource",
+                    "hydra_core",
+                    "building",
+                    {
+                        "id": "object_id",
+                        "build_year": "building_build_year"
+                    },
+                    "geometry",
+                    "start",
+                    "end"
+                ],
+                "buildyear": [
+                    "geoblocks.geometry.base.GetSeriesBlock",
+                    "buildings",
+                    "building_build_year"
+                ],
+                "label": [
+                    "geoblocks.geometry.field_operations.Classify",
+                    "buildyear",
+                    [
+                        1900,
+                        1940,
+                        1970,
+                        1990
+                    ],
+                    [
+                        "A",
+                        "B",
+                        "C",
+                        "D",
+                        "E"
+                    ]
+                ],
+                "result": [
+                    "geoblocks.geometry.base.SetSeriesBlock",
+                    "buildings",
+                    "label_value",
+                    "label"
+                ]
+            },
+            "name": "result"
+        }
+    }
+
+Geometry output
```

Comment:
This paragraph contains info that you already included above. Can be omitted

Reply:
Leaving it in for now, the actual text was written by Martijn Krol. Going to discuss with him how to better communicate these options.